src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -66,14 +71,14 @@ public void parse_allFieldsPresent_success() {
 
         // whitespace only preamble
         assertParseSuccess(parser, PREAMBLE_WHITESPACE + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
-                + NRIC_DESC_BOB + DOB_DESC_BOB + GENDER_DESC_BOB + ADDRESS_DESC_BOB,
-                new AddCommand(expectedPerson));
+                + NRIC_DESC_BOB + DOB_DESC_BOB + GENDER_DESC_BOB + ADDRESS_DESC_BOB + PRIORITY_DESC_BOB,
+                           new AddCommand(expectedPerson));
     }
 
     @Test
     public void parse_repeatedNonTagValue_failure() {
         String validExpectedPersonString = NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + NRIC_DESC_BOB
-                + DOB_DESC_BOB + GENDER_DESC_BOB + ADDRESS_DESC_BOB;
+                + DOB_DESC_BOB + GENDER_DESC_BOB + ADDRESS_DESC_BOB + TAG_DESC_FRIEND;
```

Comment:
Similarly, I believe we can remove `TAG_DESC_FRIEND` here.

Though, we might have to rename this test case since `Tag` has its own standalone command